WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -220,6 +248,32 @@ class AppSettingsViewController: UITableViewController {
         }
     }
 
+    func imageOptimizationChanged() -> (Bool) -> Void {
+        return { [weak self] value in
+            MediaSettings().imageOptimizationEnabled = value
+
+            // Track setting changes
+            WPAnalytics.track(.appSettingsOptimizeImagesChanged, properties: ["enabled": value])
+
+            // Show/hide image optimization settings
+            guard let self = self, let tableView = self.tableView else {
```

Comment:
More Swift syntactic sugar...

```suggestion
            guard let self, let tableView else {
```

Reply:
That's interesting, I didn't know that Swift would automatically do this automatically, smart!

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/21981/commits/b71b7f88ceea2abba57d36015fe8faf7e596dca0.

Reply:
> I didn't know that Swift would automatically do this automatically, smart!

It's a [relatively new feature](https://github.com/apple/swift-evolution/blob/main/proposals/0345-if-let-shorthand.md)!